### DISCUSSION TITLE
Recommended/sane options for playing on emulator

### DISCUSSION THREAD

**Discussion by ghost** [DarkMatterCore/nxdumptool#165] [Q&A] [answered]
I am currently dumping my switch library to use with emulators, and I am trying to figure out which options make sense. I found the old guide by the DMCA'd emulator in an archive, but after the rewrite, these options aren't all present. So what are the correct settings for dumping games/updates/dlc with the rewrite? I will not be playing the dumped games on the switch, only in an emulator.

**Comment by Whovian9369** [comment, marked as answer]
The default settings will work fine.
